CHANGELOG.md

```diff
@@ -221,6 +243,8 @@ The following are deprecated and will be removed in a future release. For anythi
 - SunLighting.ambientOcclusionEnabled deprecated since version 4.27. Ambient occlusion is automatically shown and this property has no effect.
 - SunLighting.waterReflectionEnabled deprecated since version 4.27. Reflections are automatically shown and this property has no effect.
 - Swipe.iconClass deprecated since 4.27. Use icon instead.
+- TableList.multipleSelectionEnabled depricated since 4.29. Use selectionMode instead.
+- TableList.selectionEnabled depricated since 4.29. Use dragEnabled and selectionMode instead.
```

Comment:
```suggestion
- TableList.multipleSelectionEnabled deprecated since 4.29. Use selectionMode instead.
- TableList.selectionEnabled deprecated since 4.29. Use dragEnabled and selectionMode instead.
```